build.sbt

```diff
@@ -334,98 +347,101 @@ lazy val chisel = (project in file("."))
   .dependsOn(macros)
   .dependsOn(core)
   .dependsOn(firrtl)
-  .aggregate(macros, core, plugin, firrtl)
+  .dependsOn(svsim)
```

Comment:
why does chisel the project depend on svsim? Shouldn't this go in the chisel test project dependencies, if anywhere today?

Reply:
(also I don't really understand how this file works, so my question may be nonsensical)